public/v4/apps/emailengine.yml

```diff
@@ -0,0 +1,44 @@
+captainVersion: 4
+services:
+  # EmailEngine
+  $$cap_appname:
+    depends_on:
+      - $$cap_appname-redis
+    image: andris9/emailengine:$$cap_emailengine_version
+    restart: always
+    environment:
+      WAIT_HOSTS: srv-captain--$$cap_appname-redis:6379
+      EENGINE_REDIS: redis://:@srv-captain--$$cap_appname-redis:6379
```

Comment:
Please test these final changes and make sure it can be deployed. I'm not sure if `redis://:@` is a valid schema...

Reply:
It's work, I test it now.